### ISSUE TITLE
screen is floating in the center of the monitor

### ISSUE THREAD

**Issue by AlanRibas** [libretro-mirrors/beetle-saturn-libretro#167]
Whenever I start a game, its screen is floating in the center of the monitor.
Is there an option to make the image touch the upper and lower edges of the monitor? But without stretching the image? Keeping the original 4: 3 aspect ratio.

![20200605_005420](https://user-images.githubusercontent.com/16063712/83836044-c65ee800-a6c8-11ea-8a64-c31de04c835c.jpg)

**Comment by AlanRibas** [comment]
I got the effect I wanted by raising the line values (circled in red).
however I did not find the option to save the configuration in a standard way for all games. But only for the game itself that is running.
Is that even possible?
specifically modify these line values and apply to any game that starts with that core?
Because I tried to start only the core and make these settings but the options to change these lines do not appear when there is no game started.

![20200607_210019](https://user-images.githubusercontent.com/16063712/83983281-5ab48f00-a903-11ea-8c53-6dbca87be247.jpg)
